src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,22 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.3.0
+
+### New Feature
+
+- Git Catalog to provide predefined development environments to every developer
+- Add Git Catalog tab to Okteto UI deploy
+- Remove CLI tab from Okteto UI deploy
```

Comment:
```suggestion
- Remove CLI tab from Okteto UI deploy
```

Do you mean the "Deploy" section of the UI? If yes I think it would be better to frame it as: "Remove CLI tab from the Deploy section of Okteto UI" or maybe just leave it at: "Remove CLI tab from Okteto UI"

Reply:
is this something we necessarily have to include in the notes?

Reply:
Yeah, otherwise people might think that it's a bug or a misconfiguration. What's your concern @rlamana ?

Reply:
No concern, just wondering. It makes sense, like you said it's a way to notify that something went away for a reason. I was just thinking about the "what" and "why", as the discussed in other comments, and wondering how to explain this one. I guess it would be: "to simplify and offer a better and seamless deploy experience in the UI".